image/v3/modifying/text-fonts.md

```diff
@@ -73,9 +73,9 @@ Define a font size. By default a value of `12` will be applied.
 
 ### Font file
 
-> public FontInterface::filename(string $filename): FontInterface
+> public FontInterface::setFilename(string $filename): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.